std_msgs/Header header

uint8 driving_direction_unconfirmed
uint8 driving_direction_confirmed